### DISCUSSION TITLE
Photoprism doesnt scan shared folder from host OS

### DISCUSSION THREAD

**Discussion by amgschnappi** [photoprism/photoprism#3395] [Q&A]
#### 1. What is not working as documented?

This example/friends is not working-

volumes:
  - "~/Pictures:/photoprism/originals"
  - "/example/friends:/photoprism/originals/friends"
  
#### 2. How can we reproduce it?

Steps to reproduce the behaviour:

1. Install Ubuntu as host OS on your laptop.
2. Copy many photos in its Pictures folder.
3. Install VirtualBox
4. Install elementary OS in virtualbox as guest OS
5. Share Ubuntu/home/Pictures folder with eOS/home/guest_picts folder using Device> Shared folder etc of VirtualBox. I can do a "cd /home/user_ubuntu/Guest_Folder/Pictures" to that guest_picts folder in eOS.
6. in elementary OS, install docker rootless
7. Download docker compose yml for Photoprism
8. Update it so that there is entry under volume (without quotes) "/home/user_ubuntu/Guest_Folder/Pictures:/photoprism/originals/from_ubuntu"
9. docker-compose up
10. Login to photoprism local website and index

#### 3. What behavior do you expect?

Pictures from ubuntu-Pictures folder should be indexed 

#### 4. What could be the cause of your problem?

Dont know

#### 5. Can you provide us with example files for testing, error logs, or screenshots?

Generic issue. Sample data not needed

#### 6. Which software versions do you use?

(a) PhotoPrism Architecture & Build Number: AMD64, image: photoprism/photoprism:latest

(b) Database Type & Version: [MariaDB, MySQL, SQLite,...](image: mariadb:10.11)

(c) Operating System Types & Versions: Linux

(d) Browser Types & Versions: Firefox

(e) Ad Blockers, Browser Plugins, and/or Firewall Software?
No

#### 7. On what kind of device is PhotoPrism installed?

Laptop

(a) Device / Processor Type: Intel Core i3, 

#### 8. Do you use a Reverse Proxy, Firewall, VPN, or CDN?
No

**Comment by graciousgrey** [comment]
Do you have cgroupfs-mount installed as described here  https://docs.photoprism.app/getting-started/troubleshooting/docker/#overlay-volumes ?
